docs/index.asciidoc

```diff
@@ -243,10 +243,10 @@ Refer to <<plugins-{type}s-{plugin}-ecs_metadata,ECS mapping>> for detailed info
 ===== `executor_threads`
 
   * Value type is <<number,number>>
-  * Default value is 1 executor thread per CPU core
+Default value is equal to the number of CPU cores.
```

Comment:
Default value is equal to the number of CPU cores (1 executor thread per CPU core).